apps/cookbook/src/app/showcase/list-showcase/list-showcase.component.html

```diff
@@ -21,7 +21,11 @@ <h2>With items</h2>
 
 <div class="page-example">
   <div>
-    <h2>List with bold text on row selection</h2>
+    <h2>List with selectable items</h2>
+    <p>
+      The selected item is highligted in bold by default. This behaviour can be suppressed by
+      passing <code>[disableSelectionHighlight]="true"</code> to the list component.
+    </p>
```

Comment:
With the addition of your nice description, I honestly think this first example is unnecessary. We now have the 'List with selectable items and dividers' example that explain things nicely, so this seems rendundant and confusing. Should we consider removing this example entirely, while we are at it?

Reply:
Good point - example removed